### DISCUSSION TITLE
Welcome to DSA-Questions Discussions!

### DISCUSSION THREAD

**Discussion by avinash201199** [avinash201199/DSA-Questions#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nitinkumar30** [comment]
I met with a question about printing square of a number in C using DS. Maybe Linked List can be used. But I want to know logic how to proceed. I guess this is a great platform for asking.

**Reply by avinash201199** [reply]
> I met with a question about printing square of a number in C using DS. Maybe Linked List can be used. But I want to know logic how to proceed. I guess this is a great platform for asking.

yeah but people are not active on discussion tab , I will try to answer your question soon
